rp_yal/views.py

```diff
@@ -22,10 +25,8 @@ def post(self, request, org_id):
             )
 
         if not org.users.filter(id=request.user.id).exists():
-            return JsonResponse(
-                data={
-                    "error": "Authenticated user does not belong to specified Organization"
-                },
+            return json_error(
+                +"Authenticated user does not belong to specified Organization",
```

Comment:
Not sure what the `+` is for here? Also why the separate `json_error` function?

Reply:
Readability, mostly:
```python
return json_error(
    "msg"
)
```
vs
```python
return JsonResponse(
    {
        "error": "msg",
    },
    status=status.HTTP_400_BAD_REQUEST,
)
```
If I recall correctly, we did the same thing in another file with a lot more `JsonResponse`s. (Some of the responses had long lines that needed to be broken up anyway, and removing an indent level and dict key helped with that too.)

Reply:
Cool, maybe we can put the `json_error` function somewhere and just reuse it from there.

And the `+`?